imagetagger/imagetagger/settings.py

```diff
@@ -0,0 +1,239 @@
+        messages.SUCCESS: 'success',
+    }
+
+    # Static files (CSS, JavaScript, Images)
+    # https://docs.djangoproject.com/en/1.10/howto/static-files/
+
+    STATIC_URL = '/static/'
+
+    EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
+
+    ########################################################################
+    #
+    # Custom Imagetagger defined settings
+    #
+    ########################################################################
+    UPLOAD_NOTICE = 'By uploading images to this tool you accept that the images get published under creative ' \
+                    'commons license and confirm that you have the permission to do so.'
+
+    EXPORT_SEPARATOR = '|'
+
+    IMAGE_PATH = 'images'  # the path to the folder with the imagesets relative to the filesystem root (see FS_URL)
+    TMP_IMAGE_PATH = 'images'  # the path to use for temporary image files relative to the temp filesystem (see TMP_FS_URL)
+    TOOLS_PATH = 'tools'  # the path to the folder with the tools relative to the filesystem root (see FS_URL)
+
+    # filename extension of accepted imagefiles
+    IMAGE_EXTENSION = {
+        'png',
+        'jpeg',
+    }
+
+    # Sets the default expire time for new messages in days
+    DEFAULT_EXPIRE_TIME = 7
+
+    # Sets the default number of messages per page
+    MESSAGES_PER_PAGE = 10
+
+    ACCOUNT_ACTIVATION_DAYS = 7
+
+    ########################################################################
+    #
+    # Computed properties and hooks
+    #
+    ########################################################################
+    @property
+    def DATABASES(self):
+        """https://docs.djangoproject.com/en/1.10/ref/settings/#databases"""
+        return {
+            'default': {
+                'ENGINE': 'django.db.backends.postgresql_psycopg2',
+                'HOST': self.DB_HOST,
+                'PORT': self.DB_PORT,
+                'NAME': self.DB_NAME,
+                'USER': self.DB_USER,
+                'PASSWORD': self.DB_PASSWORD
+            }
+        }
+
+    @classmethod
+    def post_setup(cls):
+        super().post_setup()
+
+        if cls.SENTRY_REPORTING_ENABLED:
+            try:
+                import sentry_sdk
+                from sentry_sdk.integrations.django import DjangoIntegration
+                sentry_sdk.init(
+                    dsn=cls.SENTRY_DSN,
+                    integrations=[DjangoIntegration()],
+                    # If you wish to associate users to errors you may enable sending PII data.
+                    send_default_pii=cls.SENTRY_SEND_DEFAULT_PII
+                )
+            except ImportError:
+                raise ImproperlyConfigured("Could not import sentry although the server is configured to use it")
+
+    ########################################################################
+    #
+    # User-adaptable settings
+    #
+    ########################################################################
```

Comment:
How are these different from Custom ImageTagger defined settings?

Reply:
They are not. They are properly configurable since 696b89fcedc12ce9b3a588363b8c112b9dcd98bf